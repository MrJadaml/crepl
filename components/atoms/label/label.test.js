import React from 'react';
import Label from './index';
import { cleanup, render } from '@testing-library/react';
import { getByText } from '@testing-library/dom';

afterEach(cleanup);

describe('Label', () => {
  it('should render a label', () => {
    const { asFragment } = render(
      <Label htmlFor="foo">
        Foo label
      </Label>
    );

    expect(asFragment()).toMatchSnapshot();
  });

  it('should render children', () => {
    const { getByText } = render(
      <Label htmlFor="bar">
        Bar label
      </Label>
    );

    getByText('Bar label');
  });
});
