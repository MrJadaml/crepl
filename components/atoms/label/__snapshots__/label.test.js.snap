// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should render a label 1`] = `
<DocumentFragment>
  <label
    class="label"
    for="foo"
  >
    Foo label
  </label>
</DocumentFragment>
`;
